data/data.json

```diff
@@ -8822,6 +8822,49 @@
    "aciGbpServerContainer": "noiro/gbp-server:5.1.1.0.1ae238a",
    "aciOpflexServerContainer": "noiro/opflex-server:5.1.1.0.1ae238a"
   },
+  "v1.22.3-rancher1-1": {
+   "etcd": "rancher/mirrored-coreos-etcd:v3.5.0",
+   "alpine": "rancher/rke-tools:v0.1.78",
+   "nginxProxy": "rancher/rke-tools:v0.1.78",
+   "certDownloader": "rancher/rke-tools:v0.1.78",
+   "kubernetesServicesSidecar": "rancher/rke-tools:v0.1.78",
+   "kubedns": "rancher/mirrored-k8s-dns-kube-dns:1.17.4",
+   "dnsmasq": "rancher/mirrored-k8s-dns-dnsmasq-nanny:1.17.4",
+   "kubednsSidecar": "rancher/mirrored-k8s-dns-sidecar:1.17.4",
+   "kubednsAutoscaler": "rancher/mirrored-cluster-proportional-autoscaler:1.8.3",
+   "coredns": "rancher/mirrored-coredns-coredns:1.8.4",
+   "corednsAutoscaler": "rancher/mirrored-cluster-proportional-autoscaler:1.8.3",
+   "nodelocal": "rancher/mirrored-k8s-dns-node-cache:1.18.0",
+   "kubernetes": "rancher/hyperkube:v1.22.3-rancher1",
+   "flannel": "rancher/mirrored-coreos-flannel:v0.15.1",
+   "flannelCni": "rancher/flannel-cni:v0.3.0-rancher6",
+   "calicoNode": "rancher/mirrored-calico-node:v3.20.2",
+   "calicoCni": "rancher/mirrored-calico-cni:v3.20.2",
+   "calicoControllers": "rancher/mirrored-calico-kube-controllers:v3.20.2",
+   "calicoCtl": "rancher/mirrored-calico-ctl:v3.20.2",
+   "calicoFlexVol": "rancher/mirrored-calico-pod2daemon-flexvol:v3.20.2",
+   "canalNode": "rancher/mirrored-calico-node:v3.20.2",
+   "canalCni": "rancher/mirrored-calico-cni:v3.20.2",
+   "canalControllers": "rancher/mirrored-calico-kube-controllers:v3.20.2",
+   "canalFlannel": "rancher/mirrored-coreos-flannel:v0.15.1",
```

Comment:
@kinarashah we forgot to bump this in the previous PR, so this was the only additional change I made to 1.22 template. It was still using the older version of Flannel for canal before.

https://github.com/rancher/kontainer-driver-metadata/blob/ce2fd05f6585277b8273e2d5171c8c2f660214a6/rke/k8s_rke_system_images.go#L6111

Reply:
Thanks, I missed that flannel is used for canal images too.